packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.test.tsx

```diff
@@ -110,14 +120,6 @@ describe('SearchBox', () => {
     expect(onFocus.mock.calls.length).toBe(1);
   });
 
-  it('generates id internally and cannot be set via props', () => {
```

Comment:
@KevinTCoughlin Noticed after making this PR that you wrote this test to specifically prevent use of a passed-in ID in #6477--why was that?

Reply:
Only because it was maintaining its own reference via state. However, it doesn't look like the ID is used as a reference for an ARIA-* attributes or the like, so the change seems safe.